appinventor/blocklyeditor/src/typeblock.js

```diff
@@ -547,7 +547,7 @@ Blockly.TypeBlock.prototype.createAutoComplete_ = function(inputText){
             canonicName: 'text',
             dropDown: {
               titleName: 'TEXT',
-              value: blockName.substring(1)
+              value: blockName.endsWith('"') ? blockName.substring(1, blockName.length - 1) : blockName.substring(1)
```

Comment:
Since this would require us to bump the compatibility matrix, we may want to use `goog.string.endsWith` instead.

Reply:
@ewpatton sorry for the long delay. I changed that can you please review it? Thanks